packages/client/components/RetroGroupPhase.tsx

```diff
@@ -55,6 +55,9 @@ const RetroGroupPhase = (props: Props) => {
         id
         endedAt
         showSidebar
+        autogroupReflectionGroups {
+          groupTitle
```

Comment:
-1 I think you should use the fragment `GenerateGroupsFrag_meeting` here (you could even just put it in this file) to indicate these need to match for the update.

Reply:
I can't use `GenerateGroupsFrag_meeting` here as it's called on the `GenerateGroupsSuccess` type. Perhaps I could create the fragment within the component to make it clearer? Something like this?

```
  graphql`
    fragment RetroGroupPhaseGenerateGroups_meeting on RetrospectiveMeeting {
      autogroupReflectionGroups {
        groupTitle
      }
    }
  `
  const meeting = useFragment(
    graphql`
      fragment RetroGroupPhase_meeting on RetrospectiveMeeting {
        ...StageTimerControl_meeting
        ...StageTimerDisplay_meeting
        ...GroupingKanban_meeting
        id
        endedAt
        showSidebar
        ...RetroGroupPhaseGenerateGroups_meeting
        organization {
          tier
          featureFlags {
            suggestGroups
          }
        }
      }
    `,
    meetingRef
  )
```